client/src/modules/TeamDistribution/pages/TeamDistributions/TeamDistributions.tsx

```diff
@@ -0,0 +1,60 @@
+import { useMemo, useState } from 'react';
+import { PlusOutlined } from '@ant-design/icons';
+import { PageLayout } from 'components/PageLayout';
+import { isCourseManager } from 'domain/user';
+import { CoursePageProps } from 'services/models';
+import { TeamDistributionApi, TeamDistributionDto } from 'api';
+import { TeamDistributionModal } from 'modules/TeamDistribution/components/TeamDistributionModal/';
+import { Button } from 'antd';
+import { useAsync } from 'react-use';
+import { TeamDistributionCard } from 'modules/TeamDistribution/components/TeamDistributionCard';
+
+const teamDistributionApi = new TeamDistributionApi();
+
+function TeamDistributions({ session, course }: CoursePageProps) {
+  const [teamDistribution, setTeamDistribution] = useState<Partial<TeamDistributionDto> | null>(null);
+  const [distributions, setDistributions] = useState<TeamDistributionDto[]>([]);
+  const isManager = useMemo(() => isCourseManager(session, course.id), [session, course.id]);
+
+  const { loading } = useAsync(async () => {
+    const { data } = await teamDistributionApi.getCourseTeamDistributions(course.id);
+    setDistributions(data ?? []);
+  }, []);
```

Comment:
```suggestion
  }, [course.id]);
```
Please do not forget about deps array in hooks

Reply:
fixed